src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -15,7 +15,64 @@ void EnIk_Destroy(Actor* thisx, PlayState* play);
 void EnIk_Update(Actor* thisx, PlayState* play);
 void EnIk_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+void EnIk_Thaw(EnIk* this, PlayState* play);
+s32 EnIk_IsChangingAction(EnIk* this, PlayState* play);
+s32 EnIk_ChooseAttack(EnIk* this);
+void EnIk_CheckActions(EnIk* this, PlayState* play);
+void EnIk_SetupIdle(EnIk* this);
+void EnIk_Idle(EnIk* this, PlayState* play);
+void EnIk_SetupWalk(EnIk* this);
+void EnIk_WalkTowardsPlayer(EnIk* this, PlayState* play);
+void EnIk_SetupRun(EnIk* this);
+void EnIk_RunTowardsPlayer(EnIk* this, PlayState* play);
+void EnIk_SetupVerticalAttack(EnIk* this);
+void EnIk_VerticalAttack(EnIk* this, PlayState* play);
+void EnIk_SetupTakeOutAxe(EnIk* this);
+void EnIk_TakeOutAxe(EnIk* this, PlayState* play);
+void EnIk_SetupHorizontalDoubleAttack(EnIk* this);
+void EnIk_HorizontalDoubleAttack(EnIk* this, PlayState* play);
+void EnIk_SetupSingleHorizontalAttack(EnIk* this);
+void EnIk_SingleHorizontalAttack(EnIk* this, PlayState* play);
+void EnIk_SetupEndHorizontalAttack(EnIk* this);
+void EnIk_EndHorizontalAttack(EnIk* this, PlayState* play);
+void EnIk_SetupBlock(EnIk* this);
+void EnIk_Block(EnIk* this, PlayState* play);
+void EnIk_SetupReactToAttack(EnIk* this, s32 arg1);
+void EnIk_ReactToAttack(EnIk* this, PlayState* play);
+void EnIk_SetupDie(EnIk* this);
+void EnIk_Die(EnIk* this, PlayState* play);
+void EnIk_Frozen(EnIk* this, PlayState* play);
+void EnIk_SetupCutscene(EnIk* this);
+void EnIk_PlayCutscene(EnIk* this, PlayState* play);
+void EnIk_UpdateDamage(EnIk* this, PlayState* play);
+void EnIk_UpdateArmor(EnIk* this, PlayState* play);
+
+typedef struct {
+    /* 0x0 */ Gfx* unk00;
+    /* 0x4 */ s16 unk04;
+} EnIkUnkStruct; // size = 0x6
```

Comment:
```suggestion
} EnIkUnkStruct; // size = 0x8
```
Due to alignment, this would be size 0x8